controllers/api/booking-routes.js

```diff
@@ -1,8 +1,91 @@
-// http://url/api/booking/*
 const router = require("express").Router();
+const { Booking, User, Service } = require("../../models");
+const withAuth = require("../../utils/auth");
 
+// GET all bookings 
 
+router.get ('/', withAuth, async (req,res) => {
+    try {
+        const bookingData = await Booking.findAll({

```

Comment:
This will get all bookings in the database as it include no filtering.

Arguably, this is not needed as we are generally doing all the Get routes in the page routes and as such the gets here can be removed. 

If you wanted to fix this so it returns only the bookings for the user calling it, do something like the following after the include property: 
```
where: { client_id: req.session.user_id, }
```

Reply:
Yup, that makes sense.  Removing lines 7-36.